### DISCUSSION TITLE
Unused styles eslint rule

### DISCUSSION THREAD

**Discussion by hbriese** [jpudysz/react-native-unistyles#68] [Feature Requests]
# Features
Eslint rule that detects unused styles.
Inspired by now unmaintained [eslint-plugin-react-native](https://github.com/Intellicode/eslint-plugin-react-native/)

[See the implementation](https://github.com/Intellicode/eslint-plugin-react-native/blob/master/lib/rules/no-unused-styles.js) for detecting unused StyleSheet.create styles

**Comment by jpudysz** [comment]
Cool thing, I will explore it later!

**Comment by jpudysz** [comment]
Added to roadmap for 2.x

**Reply by efstathiosntonas** [reply]
@jpudysz is there any progress in this one? I'm tired using AI to [sort ](https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/sort-styles.md)the styles 😅

**Reply by jpudysz** [reply]
I'm back after Easter holidays and I'm ready to push it forward. Nevertheless bugs have bigger priority
